e2e-tests/test-display-themes.spec.js

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '../playwright';
+test('Verify Display Themes', async ({ page }) => {
```

Comment:
A better title could be `should be able to switch between light and dark theme`

Also @ved-bruno I believe there should be two tests here
1. Clocking on radio buttons and expecting the theme to change
2. Ensuring that the system theme config is respected `page.emulateMedia({ colorScheme: systemTheme });` which is the current test you have.